libs/labelbox/src/labelbox/schema/internal/data_row_create_upsert.py

```diff
@@ -0,0 +1,76 @@
+from abc import ABC, abstractmethod
+from typing import List, Tuple, Optional
+
+from labelbox.schema.identifiable import UniqueId, GlobalKey
+from labelbox.pydantic_compat import BaseModel
+
+
+class DataRowItemBase(BaseModel, ABC):
```

Comment:
This class does not look to be abstract - all methods are implemented.

I'm not sure you ultimately need/want inheritance, perhaps we can take advantage of another OO design pattern.